grails-app/controllers/org/pih/warehouse/user/UserController.groovy

```diff
@@ -445,29 +446,8 @@ class UserController {
         User user = params.user.id ? User.get(params.user.id) : null
         Location location = params.location?.id ? Location.get(params.location.id) : null
         List<Role> roles = params.list("role.id").collect { roleId -> Role.get(roleId) }
-
-        // Update existing role
         LocationRole locationRole = LocationRole.get(params.id)
```

Comment:
This is probably ok for now, but we should be careful about how we refactor these actions. I don't actually know what the original code is trying to do (I'm to blame as the original author) so adding a method with a bunch of arguments might not be the right solution. 

In order to refactor properly 

1. We need to be very thoughtful about what the interface / API is supposed to be doing here. For example, are we trying to do multiple actions? If so, maybe we need multiple methods.
1. Maybe think about passing each param separately and loading objects in the service that way we're doing all reading / writing in the same Hibernate session.

cc @awalkowiak @alannadolny @kchelstowski

Reply:
We also need to consider the return value for new service methods. Again this is why I'd prefer to leave controllers alone unless we have very straightforward refactorings.

Reply:
This import was removed in the PR mentioned above.